src/main/scala/com/typesafe/conductr/sbt/console/Columns.scala

```diff
@@ -105,7 +105,7 @@ object Column {
       bundles.map { bundle =>
         bundle.bundleInstallations.map { node =>
           val nodeIpAndPort = node.uniqueAddress.address.toString.dropWhile(_ != '@').drop(1)
-          if (bundle.bundleExecutions.map(_.host) contains node.uniqueAddress.address.host.get)
+          if (bundle.bundleExecutions.filter(_.isStarted).map(_.host) contains node.uniqueAddress.address.host.get)
```

Comment:
But your solution iterates the collection twice!
Also, you can improve the function given to collect:
``` scala
collect { case BundleExecution(host, true) => host }
```